.travis.yml

```diff
@@ -52,15 +52,14 @@ matrix:
                MNE_DIRS="minimum_norm preprocessing realtime simulation stats time_frequency viz"
                OPTION=""
 
-        # Old dependencies first half
+        # Old dependencies first half (Pandas needs numexpr for eval)
         - os: linux
-          env: CONDA_DEPENDENCIES="numpy=1.8 scipy=0.12 matplotlib=1.3 pandas=0.12 scikit-learn=0.14 pytest"
+          env: CONDA_DEPENDENCIES="numpy=1.8 scipy=0.12 matplotlib=1.3 pandas=0.13 scikit-learn=0.14 numexpr pytest"
```

Comment:
hum... numexpr is in maintainance mode and it's a bit dependency. Also on pandas doc I read:

```
Note You should not use eval() for simple expressions or for expressions involving small DataFrames. In fact, eval() is many orders of magnitude slower for smaller expressions/objects than plain ol’ Python. A good rule of thumb is to only use eval() when you have a DataFrame with more than 10,000 rows.```

so can we avoid this?

Reply:
`numexpr` is only necessary for old Pandas when using eval

so if we switch to something else, `numexpr` can go away. but that's up to the Pandas lovers, I have no opinion